### ISSUE TITLE
Auro and a11y page for the docsite

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/AuroDocsSite#31]
# Auro and a11y

The Auro docsite needs a page that illustrates the opinion of how web accessibility is solved with code in Auro. 

This page will discuss how the WC-Generator has support built in. Will be hosted in the `./docs` dir and the docsite will subscript to this page. 

## What's in the page? 

> Alaska Airlines we don't know everything about accessibility but that doesn't keep us from trying every day we learn we test we learn we test and we are always striving to give our customers the best experience possible regardless of their abilities

The initial thought is to illustrate components that support a11y core patterns that would be adopted by any Auro component. 

## The docsite

There needs to be an `accessibility` section of the site with subnav. The [a11y statement](https://auro.alaskaair.com/a11y-statement) page needs to be moved to this section. 

The a11y statement content itself should be moved to the WC-Generator `./docs` dir.
